tools/appliance/systemvmtemplate/scripts/install_systemvm_packages.sh

```diff
@@ -59,19 +60,21 @@ function install_packages() {
     xenstore-utils libxenstore3.0 \
     ipvsadm conntrackd libnetfilter-conntrack3 \
     keepalived irqbalance \
-    ipcalc \
     openjdk-11-jre-headless \
-    ipset \
+    ipcalc ipset \
     iptables-persistent \
     libtcnative-1 libssl-dev libapr1-dev \
-    python-flask \
     haproxy \
     haveged \
     radvd \
     sharutils genisoimage \
     strongswan libcharon-extra-plugins libstrongswan-extra-plugins strongswan-charon strongswan-starter \
     virt-what open-vm-tools qemu-guest-agent hyperv-daemons
 
+  # python2-netaddr workaround
+  wget https://github.com/shapeblue/cloudstack-nonoss/raw/main/python-netaddr_0.7.19-1_all.deb
```

Comment:
we could use ipaddress.ip_address and ipaddress.ip_network instead of IPAddress and IPNetwork 
then python-netaddr will be not needed.

Reply:
Yes that would be separate refactoring needed to get rid of netaddr dependency; I did n't want to change too many things in this PR